### PR TITLE
Hours Spent Field Tooltip + Tooltip Question Mark refactor

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request extracts the specialized 'question mark' Tooltip from the `CampaignInfoBlock` into its own component, refactors it, and uses it to add a tooltip to the 'Hours Spent' field in the `PhotoSubmissionAction`.

### How should this be reviewed?
You can commit-by-commit to see the updates atomically.

### Any background context you want to provide?
We've been getting a lot of Zendesk tickets and incorrect input for the Hours Spent field so along with #2606 to simplify the field, the tooltip should hopefully help ensure correct submissions!

### Relevant tickets

References [Pivotal #177565416](https://www.pivotaltracker.com/story/show/177565416).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [ ] Added screenshots of front-end changes on small, medium, and large screens.
- [ ] Added appropriate feature/unit tests.

image

### CODE REVIEW

File: resources/assets/components/actions/PhotoSubmissionAction/HoursSpentField.js [hunk 1]
```diff
@@ -34,6 +35,7 @@ const HoursSpentField = ({ actionId, hasError, onChange }) => {
               })}
             >
               How long did this action take?
+              <TooltipQuestionMark tooltipContent="The number of hours you report should reflect the time it took you to complete this action. Please ensure that the number of hours you are reporting is accurate or your post will be rejected." />
```
Comment (DFurnes) [inline resources/assets/components/actions/PhotoSubmissionAction/HoursSpentField.js:38]: Language nitpick: would it make more sense to say "or your post _may_ be rejected"? To me, the wording "will be rejected" feels aggressive/confrontational.

**Review by DFurnes — APPROVED** [review]
This looks great! 👏

File: resources/assets/components/actions/PhotoSubmissionAction/HoursSpentField.js (hunk 1 above)
Comment (mendelB) [reply resources/assets/components/actions/PhotoSubmissionAction/HoursSpentField.js:38]: Hmm, I agree! I copied this over from the document, but I have a strong feeling Leah will agree as well -- I'll tag her on Slack!

Comment (mendelB) [reply resources/assets/components/actions/PhotoSubmissionAction/HoursSpentField.js:38]: Gahh, some more context from Leah:

> I'd keep with the stronger language - we have had a very HIGH amount of people (over 10% of posts) saying ridiculous amounts of hours (i.e., 20 hours for 5 cans collected). When in doubt, we do accept the reportback, but clearly the youngs are trying to be sneaky!

Looks like we want to maintain this stronger language!

**Pull request merged**